src/components/Button/Button.tsx

```diff
@@ -1,59 +1,147 @@
 import React, { FC } from "react";
+import styled, { css } from "styled-components";
+import { Color } from "../../theme/util";
 
 // https://www.w3schools.com/tags/tag_button.asp
 // https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role
 export interface ButtonProps {
   ariaExpanded?: boolean | undefined;
   ariaPressed?: boolean | "mixed" | undefined;
+  ariaLabel?: string | undefined;
   autoFocus?: boolean;
   children: string | React.ReactNode;
   className?: string;
   disabled?: boolean;
   name?: string;
-  role: string;
```

Comment:
Why is the `role` prop remove? We should keep it for screen readers.

Reply:
Sorry, I thought I commented that in the code. 

I removed role because role doesn't accept custom strings like "primary button" and fail accessibility checks. I did some research and we only should have role defined if we are using a link or non-button to act as a button, which this component won’t do. It seems if we use html's <button> the role is already button 
      https://dequeuniversity.com/rules/axe/3.5/aria-roles?application=axeAPI
     https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/button/button.html

Reply:
But maybe this prop didn't need to be removed.. I will do some more reading. But either way there is a list of approved roles that we should use in the first link, although for this case it would always be button

Reply:
Cool, I didn't know this. Thanks!

Reply:
ok, it sounds like we either use role="button" on something that's not a button, or the semantic <button>, so for now I don't think we need the prop https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role